apps/tenant-management-webapp/src/app/pages/admin/services/form/definitions/definitions.tsx

```diff
@@ -98,15 +98,29 @@ export const FormDefinitions = ({ openAddDefinition }: FormDefinitionsProps) =>
       />
 
       {!indicator.show && !formDefinitions && renderNoItem('form templates')}
-      {indicator.show && <PageIndicator />}
-      {!indicator.show && Object.keys(formDefinitions).length > 0 && (
-        <FormDefinitionsTable
-          definitions={formDefinitions}
-          onDelete={(currentTemplate) => {
-            setShowDeleteConfirmation(true);
-            setCurrentDefinition(currentTemplate);
-          }}
-        />
+      {/* {indicator.show && <PageIndicator />} */}
+      {formDefinitions && Object.keys(formDefinitions).length > 0 && (
+        <>
+          <FormDefinitionsTable
+            definitions={formDefinitions}
+            onDelete={(currentTemplate) => {
+              setShowDeleteConfirmation(true);
+              setCurrentDefinition(currentTemplate);
+            }}
+          />
+          {next && (
+            <LoadMoreWrapper>
+              <GoAButton
+                testId="calendar-event-load-more-btn"
```

Comment:
@RamaChandiniDevarapalli  please update the testids in your html that has calendar in it so it doesnt break the e2e.